app/views/feedbacks/_main_fields.html.slim

```diff
@@ -0,0 +1,12 @@
+
+= f.hidden_field :volunteer_id, value: @feedback.volunteer.id
+= f.hidden_field :author_id, value: @feedback.author_id || current_user.id
```

Comment:
- the `value:` shouldn't really be necessary since it's already taken from `@feedback` by default
- you're already assigning `author` in the controller so `current_user` shouldn't be necessary here
- is it actually necessary to send `author_id` in the form, i.e. can a user create feedback for another user, or change the author of a feedback? If not, you could/should remove `author_id` from the form and `feedback_params` and only set the author in the `create` action

Reply:
> the value: shouldn't really be necessary since it's already taken from @feedback by default
you're already assigning author in the controller so current_user shouldn't be necessary here

Seems to make quite some sense 👍 

> is it actually necessary to send author_id in the form, i.e. can a user create feedback for another user, or change the author of a feedback? If not, you could/should remove author_id from the form and feedback_params and only set the author in the create action

the superadmins can create feedback on a volunteers GroupOffer or Assignment. Thats why there is a author field on Feedback at all, because if only the volunteer.user could create feedback on its own stuff, the field would render obsolete.